CHANGELOG.rst

```diff
@@ -17,6 +17,17 @@ Unreleased
 ----------
 * Nothing
 
+[6.0.0]
+
+* We no longer test with ubuntu 20.04 and test with whatever is the latest
+  ubuntu supplied by the ``ubuntu-latest`` github action.  Not actually a
+  breaking change but a change in expectations that we're noting here.
+
+Breaking Changes
+****************
+
+* Dropping Python 3.8 support
```

Comment:
Are there any questions around timing of this? Are all services using this library ready for this to be dropped? If not, will this unnecessarily cause issues if someone wants to update this library and they can’t bring in the change? Apologies if I am behind the times and we’re dropping 3.8 everywhere at this time.

Reply:
We're dropping 3.8 everywhere, it's out of support upstream next month.